### PR TITLE
[WIP] Attempted to update PHP WinCache documentation.

### PR DESCRIPTION
**Pull request by lextm**
The initial PR fixed two links.

### CODE REVIEW

File: iis/application-frameworks/scenario-build-a-php-website-on-iis/configuring-step-1-install-iis-and-php.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ To keep this procedure simple, install the WinCache extension but do not configu
 ### To download and install PHP and WinCache
 
 1. Open your browser to [Windows for PHP Download Page](http://windows.php.net/download/) and download the PHP non-thread-safe zip package.
-2. Download the WinCache extension from the [List of Windows Extensions for PHP](http://downloads.php.net/pierre/).
+2. Download the WinCache extension from the [List of Windows Extensions for PHP](http://windows.php.net/downloads/pecl/releases/wincache/).
```
Comment (bariscaglar) [inline iis/application-frameworks/scenario-build-a-php-website-on-iis/configuring-step-1-install-iis-and-php.md]: https://sourceforge.net/projects/wincache/ seems to be the more up to date location for download.

File: iis/application-frameworks/scenario-build-a-php-website-on-iis/planning-step-1-plan-iis-web-server-and-php-installation.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ There are two builds for each PHP version: one is thread-safe, and the other is
 
 PHP offers several extensions that extend its base functionality. One extension is important for PHP web applications that run on IIS-the Windows Cache Extension for PHP (WinCache). WinCache is a PHP accelerator that is used to increase the speed of PHP applications that run on Windows and Windows Server. Once the WinCache is enabled and loaded by the PHP engine, PHP applications can take advantage of the functionality without any code modifications.
 
-You can download WinCache extension from the [PHP for Windows](http://windows.php.net/) site.
+You can download WinCache extension from the [PHP for Windows](http://windows.php.net/downloads/pecl/releases/wincache/) site.
```
Comment (bariscaglar) [inline iis/application-frameworks/scenario-build-a-php-website-on-iis/planning-step-1-plan-iis-web-server-and-php-installation.md]: Same comment here.

**Comment by lextm** [comment]
Thanks. Will wait till #128 provides some idea on which download page will be the best (iis.net/php.net/sourceforge.net).

**Comment by bariscaglar** [comment]
I confirmed with Eric on a separate thread that the location we should be using is Sourceforge.

**Comment by lextm** [comment]
Got it. The links are changed to point to SourceForge.

**Pull request merged**